layers.go

```diff
@@ -563,6 +563,8 @@ func (s *store) newLayerStore(rundir string, layerdir string, driver drivers.Dri
 		uidMap:         copyIDMap(s.uidMap),
 		gidMap:         copyIDMap(s.gidMap),
 	}
+	defer rlstore.Unlock()
```

Comment:
It’s more idiomatic to call `defer` only _after_ locking the lock. Please update both.

Reply:
Oh, sure. Fixed